examples/animation/animate_decay.py

```diff
@@ -32,6 +32,6 @@ def run(data):
 
     return line,
 
-ani = animation.FuncAnimation(fig, run, data_gen, blit=True, interval=10,
+ani = animation.FuncAnimation(fig, run, data_gen, blit=False, interval=10,
```

Comment:
Wouldn't need to do that. The animation module already attaches a reset of
the blit to frame resizes, we just never attached it to other kinds of
changes.

I don't know why it doesn't make a difference for you though. A full redraw
isn't sufficient. The bbox copy has to be reset.
On Jan 8, 2015 10:32 PM, "Thomas A Caswell" <notifications@github.com>
wrote:

> In examples/animation/animate_decay.py
> <https://github.com/matplotlib/matplotlib/pull/3985#discussion-diff-22700042>
> :
>
> > @@ -32,6 +32,6 @@ def run(data):
> >
> >      return line,
> >
> > -ani = animation.FuncAnimation(fig, run, data_gen, blit=True, interval=10,
> > +ani = animation.FuncAnimation(fig, run, data_gen, blit=False, interval=10,
>
> ah, ok. If you move the mouse in and out it triggers a full re-draw (at
> least on qt) which updates them so I thought it was always updating
> correctly.
>
> There probably should be some logic someplace in the animation that if any
> has change to do a full-redraw.
>
> This actually ties back into what @efiring <https://github.com/efiring>
> and I were talking about to make pyplot behave better as well as it
> wouldn't be too hard to co-op that system to allow non-animated artists to
> make the axes as 'dirty' which would trigger a full-redraw.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/3985/files#r22700042>.
>